dc/core/ContentLoader.cpp

```diff
@@ -53,9 +54,15 @@ bool ContentLoader::load( const QString& filename,
                           const QPointF& windowCenterPosition,
                           const QSizeF& windowSize )
 {
+    put_flog( LOG_INFO, "opening: '%s'", filename.toLocal8Bit().constData( ));
+
     ContentPtr content = ContentFactory::getContent( filename );
     if( !content )
+    {
+        put_flog( LOG_WARN, "ignoring unsupported file: '%s'",
```

Comment:
Yes, this called for adaptations in the Factory code, which was incorrect. Done.